.circleci/config.yml

```diff
@@ -446,13 +391,13 @@ jobs:
 
       - <<: *upload_coverage
 
-  gpu_tests_190:
+  gpu_tests_1_10_0:
     parameters:
       test_list_file:
         type: string
         default: "/dev/non_exist"
 
-    <<: *gpu_cu111
+    <<: *gpu_cu_11_2_small_multi
```

Comment:
when do we decide to use small_multi vs medium_multi?

Reply:
small_multi has 2 gpus and medium_multi has 4 gpus. there are certain jobs which require world_size=4. hence the use of medium_multi for some jobs. E.g. See the logs here https://app.circleci.com/pipelines/github/facebookresearch/fairscale/3740/workflows/643c4a44-4ca6-4b8f-abe2-6554c6222c18/jobs/40210